database/migrations/2020_06_01_152818_create_cms_pages_table.php

```diff
@@ -19,6 +19,7 @@ public function up()
             $table->string('title')->unique();
             $table->string('slug')->unique();
             $table->text('summary')->nullable();
+            $table->boolean('is_home');
```

Comment:
I don't see any controller change, we need a / route that shows the content of the first home field.